### ISSUE TITLE
config variable $config['des_key'] is not individual when it is not configured for a container 

### ISSUE THREAD

**Issue by zem** [roundcube/roundcubemail-docker#59]
this is a security issue! In a running roundcube installation this has to be configured individiually. 

```
// this key is used to encrypt the users imap password which is stored
// in the session record (and the client cookie if remember password is enabled).
// please provide a string of exactly 24 chars.
// YOUR KEY MUST BE DIFFERENT THAN THE SAMPLE VALUE FOR SECURITY REASONS

```
however: 

```
root@c62c0e0af57f:/var/www/html/config# grep des *
config.inc.php.sample:$config['des_key'] = 'rcmail-!24ByteDESkey*Str';
defaults.inc.php:// Includes should be interpreted as PHP files
defaults.inc.php:$config['des_key'] = 'rcmail-!24ByteDESkey*Str';
defaults.inc.php:    'notes'        => 'description',
defaults.inc.php:// Interface layout. Default: 'widescreen'.
defaults.inc.php://  'widescreen' - three columns
defaults.inc.php://  'desktop'    - two columns, preview on bottom
defaults.inc.php:$config['layout'] = 'widescreen';
mimetypes.php:  'ppsm' => 'application/vnd.ms-powerpoint.slideshow.macroEnabled.12',
mimetypes.php:  'ppsx' => 'application/vnd.openxmlformats-officedocument.presentationml.slideshow',
```

looks very samplish to me. 

I would have expected the docker image to at least create a des_key if no key is provided by the administrator. I would have also expected that this switch is documented on the dockerhub site and in the README.md to this container, as this is the place peope are supposed to look to get their stuff running. 

Workaround:  I am going to place a config.php into the container via compose.yml to configure that key.

**Comment by thomascube** [comment]
When running multiple nodes, the DES key needs to be shared among all nodes. Could be solved with Docker `secrets` request in issue #46 46

**Comment by thomascube** [comment]
Please review and re-open if necessary.
